controllers/remote/cluster_cache_healthcheck_test.go

```diff
@@ -168,7 +168,8 @@ func TestClusterCacheHealthCheck(t *testing.T) {
 			g.Expect(err).NotTo(HaveOccurred())
 			l, err := net.ListenTCP("tcp", addr)
 			g.Expect(err).NotTo(HaveOccurred())
-			l.Close()
+			err = l.Close()
+			g.Expect(err).NotTo(HaveOccurred())
```

Comment:
Could inline this as it only returns the one variable, eg
```suggestion
			g.Expect(l.Close()).To(Succeed())
```